react/AddToCartButton.tsx

```diff
@@ -165,7 +167,9 @@ function AddToCartButton(props: Props) {
   }
 
   const handleAddToCart: React.MouseEventHandler = event => {
-    event.stopPropagation()
+    const shouldStopEventPropagation = onClickEventPropagation === 'disabled'
+
+    shouldStopEventPropagation && event.stopPropagation()
```

Comment:
```suggestion
    if(shouldStopEventPropagation) {
        event.stopPropagation()
    }
```

It's more clear to use if than &&.